terms/neuro/modelSystemName.json

```diff
@@ -228,6 +228,11 @@
             "description": "triple mutant strain carrying a humanized ApoE knock-in mutation, an A528T point mutation of the Sorl1 gene. and an R47H point mutantion of Trem2",
             "source": "https://www.jax.org/strain/031940"
         },
+        {
+            "const": "Spi1*rs1377416",
```

Comment:
I have some concerns with the asterisk because asterisks have special meanings at times, e.g. regular expressions.  Can it be left out?

Reply:
Thanks for catching this, didn't even think about that. I removed the asterisks from both and I'll have Juliane let the contributor know we can't use them in model names.